setup.py

```diff
@@ -37,10 +37,23 @@
 LICENSE = metadata.get('license', 'unknown')
 URL = metadata.get('url', 'http://astropy.org')
 
-# Get the long description from the package's docstring
-__import__(PACKAGENAME)
-package = sys.modules[PACKAGENAME]
-LONG_DESCRIPTION = package.__doc__
+# order of priority for long_description:
+#   (1) set in setup.cfg,
+#   (2) load README.rst,
+#   (3) package docstring
+_cfg_long_description = metadata.get('long_description', '')
+if _cfg_long_description:
+    LONG_DESCRIPTION = _cfg_long_description
+
+elif os.path.exists('README.rst'):
```

Comment:
the README sometimes has the extension of ``rst``, but equal times it is ``md``. Can we be more flexible with it?